boards/cc1352p-launchpad/board.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C)    2016 Nicholas Jackson
+ *                  2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup         boards_cc1352p_launchpad
+ * @{
+ *
+ * @file
+ * @brief           Board specific implementations for TI CC1352P LaunchPad
+ *
+ * @author          Nicholas Jackson <nicholas.jackson@griffithuni.edu.au>
+ * @author          Sebastian Meiling <s@mlng.net>
```

Comment:
You should put your name here (and in `CODEOWNERS`) - no need to attribute boilerplate.

Reply:
i added a new @autor just below, and in `CODEOWNERS`

Reply:
Or do you mean remove these rows and put only my name?

Reply:
Remove those - they don't have the board, so people shouldn't bug them if something doesn't work :wink:

Reply:
changes addressed